### PR TITLE
[zh-cn]: update translation of Number.toExponential()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15736

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/number/toexponential/index.md [hunk 1]
```diff
@@ -3,52 +3,55 @@ title: Number.prototype.toExponential()
...
+  - : 可选。一个整数，用来指定小数点后有几位数字。默认设置为完整表示该数字所需要的数字。
 
 ### 返回值
 
-一个用幂的形式 (科学记数法) 来表示{{jsxref("Number")}} 对象的字符串。小数点后以 fractionDigits 提供的值来四舍五入。如果 `fractionDigits` 参数被忽略了，小数点后的将尽可能用最多的位数来表示该数值。
-
-对数值字面量使用 `toExponential()` 方法，且该数值没有小数点和指数时，应该在该数值与该方法之间隔开一个空格，以避免点号被解释为一个小数点。也可以使用两个点号调用该方法。
-
-如果一个数值的小数位数多余 `fractionDigits` 参数所提供的，则该数值将会在 `fractionDigits` 指定的小数位数处四舍五入。可以查看 {{jsxref("Number.toFixed", "toFixed()")}} 方法描述中关于四舍五入的讨论，同样应用于 `toExponential()` 方法。
+一个以指数记数法表示给定 {{jsxref("Number")}} 对象的的字符串，其小数点前为一位数字，小数点后舍入到 `fractionDigits` 位。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/number/toexponential/index.md:26]: exponetial notation 翻译前后统一下

**Review by jasonren0403 — COMMENTED** [review]
有一处建议

File: files/zh-cn/web/javascript/reference/global_objects/number/toexponential/index.md (hunk 1 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/number/toexponential/index.md:26]:
```suggestion
一个以指数表示法表示给定 {{jsxref("Number")}} 对象的的字符串，其小数点前为一位数字，小数点后舍入到 `fractionDigits` 位。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**